# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vdr-cdda/cdda-0.1.0-1.3.38.patch
# Copyright (C) 2006 - 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- cdda-0.1.0/audio_player.c.orig	2005-12-23 10:24:48.000000000 +0100
+++ cdda-0.1.0/audio_player.c	2006-01-15 17:28:36.000000000 +0100
@@ -368,7 +368,7 @@
 }
 
 cAudioControl::~cAudioControl() {
-  cStatus::MsgReplaying(this, NULL);
+  cStatus::MsgReplaying(this, NULL, NULL, false);
   free(mp_replayDescription);
 
   delete player;
@@ -469,7 +469,7 @@
     mp_replayDescription = p_str;
     p_str = NULL;
 
-    cStatus::MsgReplaying(this, mp_replayDescription);
+    cStatus::MsgReplaying(this, mp_replayDescription, NULL, true);
   } else
     free(p_str);
 
